django/db/backends/sqlite3/introspection.py

```diff
@@ -7,6 +8,8 @@
 )
 from django.db.models.indexes import Index
 
+FieldInfo = namedtuple('FieldInfo', FieldInfo._fields + ('pk',))
```

Comment:
I think we can import standard `FieldInfo` as a `BaseFieldInfo`, it seems to be cleaner for me (like in `oracle/introspection.py`).

Reply:
The MySQL backend has it this way... We can standardise this one way or the other. Let me know which way and I can create a PR to fix the other.

Reply:
Not sure :thinking: I would prefer to use `BaseFieldInfo` approach in this PR, and to unify introspection for PostgreSQL and MySQL in a separate PR.

Reply:
Will go with your suggestion as it seems to be the convention, cf. `BaseDatabaseIntrospection`.

Reply:
> ...and to unify introspection for PostgreSQL and MySQL in a separate PR

See #10826 for MySQL. No change required for PostgreSQL as `FieldInfo` is not overridden.